posts/2022/sparse-PFT-gridding.ipynb

```diff
@@ -0,0 +1,12504 @@
+{
```

Comment:
<p>Does this need to be a raw cell?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/esds/pull/159/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>

Reply:
<p>did it so that we can execute the notebook from top to bottom. Is there a jupyterbook tag I can set to ignore the exception if the cell is executed?</p>